client/cmd_test.go

```diff
@@ -37,10 +37,10 @@ func TestValidateCmd(t *testing.T) {
 		args    []string
 		wantErr bool
 	}{
-		{"misspelled command", []string{"comission"}, true}, // nolint: misspell
+		{"misspelled command", []string{"commission"}, true}, // nolint: misspell
 		{"no command provided", []string{}, false},
-		{"help flag", []string{"comission", "--help"}, false},       // nolint: misspell
-		{"shorthand help flag", []string{"comission", "-h"}, false}, // nolint: misspell
+		{"help flag", []string{"commission", "--help"}, false},       // nolint: misspell
```

Comment:
These are intentionally mispelled

Reply:
yea, this is misspelled in the satking module, theres a reason for it but cant remember,

Reply:
"misspelled in the satking module" :joy: